meson_options.txt

```diff
@@ -4,6 +4,7 @@ option('profile', type: 'feature', description: 'enable profiling program')
 option('fuzz', type: 'feature', description: 'enable fuzzing consumer')
 option('checkasm', type: 'feature', description: 'enable assembly unit test program')
 option('art-samples', type: 'string', description: 'Path to the root of regression testing sample repository. If set, it is used in meson test.')
+option('fuzzing-engine', type: 'string', description: 'Additional link flags for fuzz targets.')
```

Comment:
I’d prefer to neutrally name it `fuzz-ldflags` or something similar, since it can also be used for other things than the fuzzing engine. On the other side of this, from the name `fuzzing-engine` I would never expect to have to specify link flags but instead would try to name the desired engine.

Reply:
Fair, I can change that. It's kinda unfortunate we need this at all, but some meson sanity checks are generting main function, so they cannot be linked with fuzzer, which already defined it too.

Granted if we were to only support libfuzzer it could be set implicitly, but I think an option is clean and connect to oss-fuzz nicely.